cg/apps/vogue.py

```diff
@@ -60,6 +60,16 @@ def load_flowcells(self, days):
         for line in self.process.stderr_lines():
             LOG.info("vogue output: %s", line)
 
+    def load_reagent_labels(self, days):
```

Comment:
Could you add type hints? It is used more or less everywhere in CG and makes the code much easier to read. Something like
```suggestion
    def load_reagent_labels(self, days: int) -> None:
```